crates/bermuda/src/lib.rs

```diff
@@ -4,9 +4,9 @@ use pyo3::prelude::*;
 
 use triangulation::point::Triangle;
 use triangulation::{
-    split_polygons_on_repeated_edges, sweeping_line_triangulation,
-    triangulate_path_edge as triangulate_path_edge_rust, triangulate_paths_edge, PathTriangulation,
-    Point,
+    is_convex, split_polygons_on_repeated_edges, sweeping_line_triangulation,
+    triangulate_convex_polygon, triangulate_path_edge as triangulate_path_edge_rust,
+    triangulate_paths_edge, PathTriangulation, Point,
```

Comment:
@Czaki Let's improve readability by having one item per line

```rust
use triangulation::{
    is_convex,
    split_polygons_on_repeated_edges,
    sweeping_line_triangulation,
    triangulate_convex_polygon,
    triangulate_path_edge as triangulate_path_edge_rust,
    triangulate_paths_edge,
    PathTriangulation,
    Point,
};
```

Reply:
cargo format does not agree with your suggestion :(